configure

```diff
@@ -388,6 +388,7 @@ msg "inspecting environment"
 
 CFG_OSTYPE=$(uname -s)
 CFG_CPUTYPE=$(uname -m)
+ENDIAN=$(printf '\1' | od -dAn)
```

Comment:
`od` is [part of](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/od.html) POSIX (including the `-d` and `-A n` options), just as `head` and `tail` are. Is that good enough? If not, I can do your suggestion.